package CfgBoss;
import "CfgCommon.proto";


message Boss {
	required uint32 reincarnLevel= 1;
	required uint32 bossid = 2;
	required uint32 copyId = 3;
	required uint32 useType =4;
	repeated CfgCommon.Reward reward =5;
}

message Bosses {
	repeated Boss boss = 1;
}